fhir-server/src/test/java/com/ibm/fhir/server/test/CapabilitiesTest.java

```diff
@@ -24,13 +24,15 @@
 import com.ibm.fhir.model.resource.CapabilityStatement;
 import com.ibm.fhir.model.resource.CapabilityStatement.Rest.Resource.Interaction;
 import com.ibm.fhir.model.type.code.ResourceType;
+import com.ibm.fhir.search.util.SearchUtil;
 import com.ibm.fhir.server.resources.Capabilities;
 
 public class CapabilitiesTest {
 
     @BeforeClass
     void setup() {
         FHIRConfiguration.setConfigHome("target/test-classes");
```

Comment:
i think our tests are split about 50/50 between these two.  in most (all?) cases I think they should be equivalent.  was trying not to bit off too much in this already large pr

Reply:
not too worried, as long as it works the same way

Reply:
ok, in at least one case it needs to be `target/test-classes` because we do some manipulation there (e.g. renaming/moving the file) which we wouldn't want to do under src.  so i'm leaning toward making them all `target/test-classes` for consistency